src/algorithms/wcc.c

```
@@ -0,0 +1,170 @@
+
+GrB_Info WCC
+(
+	GrB_Vector *components, // [output] components
+	GrB_Vector *N,          // [output] list computed nodes
+	const Graph *g,         // graph
+	LabelID *lbls,          // [optional] labels to consider
+	unsigned short n_lbls,  // number of labels
+	RelationID *rels,       // [optional] relationships to consider
+	unsigned short n_rels   // number of relationships
+) {
+	ASSERT(g          != NULL);
```

Cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)